### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by khuongduybui**
Switch to 3.4 repo

### CODE REVIEW

**Comment by mitar** [comment]
Hm, that should be done into a `3.4` branch, no? Not `2.6`.

**Comment by mitar** [comment]
And please keep all commands in one `RUN`, so that we do not create extra Docker layers.

**Comment by mitar** [comment]
In fact, I will be changing this repository to have one branch and multiple Docker files.

Can you just create a pull request to this branch, adding a new file, `mongodb-3.4.dockerfile`? I will move others together. At the end, [it will be something like this](https://github.com/tozd/docker-runit).

**Comment by khuongduybui** [comment]
The tutorial here uses multiple command: https://docs.docker.com/engine/examples/mongodb/#building-the-mongodb-docker-image

I can make it one command as needed, but you need to create the 3.4 branch for your repo, I can't do that for you.

**Comment by mitar** [comment]
Yes, tutorial is not optimized. You can use multiple RUN commands to develop, but at the end we want one.

Branch is not needed anymore, because I decided to change the whole structure of the repository, see above.

And thanks for contributing!

**Comment by khuongduybui** [comment]
OK I made the change.
How do I make my own repo use xenial from runit instead of trusty?

**Comment by mitar** [comment]
Just use `ubuntu-xenial` to tag the `FROM` Docker image. So `FROM tozd/runit:ubuntu-xenial`.

You can also update it in this pull request if you want.

File: Dockerfile [hunk 1]
```diff
@@ -1,13 +0,0 @@
-FROM tozd/runit
```
Comment (mitar) [inline Dockerfile]: Please just add it the file, keep the old one around (or rename old one to `mongodb-2.6.dockerfile`).

**Comment by khuongduybui** [comment]
All done

**Comment by mitar** [comment]
Have you tested this file? Are you sure everything works well?

**Comment by khuongduybui** [comment]
Yes I did. Took a few tries.

**Comment by mitar** [comment]
Thanks!

**Pull request merged**